.github/workflows/linux.yml

```diff
@@ -79,19 +79,29 @@ jobs:
       - name: Cache dependencies
         uses: Swatinem/rust-cache@842ef286fff290e445b90b4002cc9807c3669641
 
-      - name: Compile
-        run: cargo build --benches --tests
+      - name: Compile (all features)
+        run: cargo build --all-features --all-targets --workspace
 
       - name: Run Rust tests (all features)
         timeout-minutes: 10
         run: |
           export TEST_GIT="$PWD"/git
           export TEST_GIT_EXEC_PATH=$(dirname "$TEST_GIT")
-          cargo test --all-features
+          cargo test --all-features --examples --tests --workspace
+
+      - name: Run Rust doc-tests (all features)
```

Comment:
Why not combine this with the test run above? You could just add `--doc` to line 90, right?

Reply:
Nope, you can't combine them for some reason 😢

Reply:
Hmm, that's surprising. I'm curious why (but I don't expect you to answer that :) )